compose.js

```diff
@@ -89,7 +89,11 @@ function mergeComposable(dstDescriptor, srcComposable) {
   combineProperty('deepConfiguration', merge);
   if (Array.isArray(srcDescriptor.initializers)) {
     if (!Array.isArray(dstDescriptor.initializers)) dstDescriptor.initializers = [];
-    dstDescriptor.initializers.push.apply(dstDescriptor.initializers, srcDescriptor.initializers.filter(isFunction));
+    srcDescriptor.initializers.forEach(i => {
```

Comment:
I agree `reduce` is always better. This file is designed to be as readable as possible. I find `reduce` less readable than the `forEach` in this case.